packages/generate-persisted-query-manifest/package.json

```diff
@@ -38,5 +38,9 @@
   "peerDependencies": {
     "@apollo/client": "^3.7.0 || ^3.8.0-alpha",
     "graphql": "14.x || 15.x || 16.x"
+  },
+  "devDependencies": {
+    "@gmrchk/cli-testing-library": "^0.1.2",
```

Comment:
nice API but fwiw it only had 1 month of development 1.5 years ago and does have open issues/PRs like https://github.com/gmrchk/cli-testing-library/pull/7 that seem potentially worrisome. But eh, we can always fork it if we have to if we like it enough.

Reply:
Absolutely! I did my best to find a technique that wouldn't require a library, but suggestions were a mess and no option seemed as elegant as this library. I didn't want to spend too much time building something myself, so I just went with this for now.

Totally understand its young and might have its own problems, but it worked well for the relatively small CLI here. Figured we could get some mileage out of it until we need more.